src/vmm/src/guest_config/aarch64_templates/mod.rs

```diff
@@ -0,0 +1,117 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use core::ops::Deref;
+use std::collections::BTreeMap;
+
+use kvm_ioctls::VcpuFd;
+
+/// Module with static aarch64 template
+pub mod g3g2;
+
+/// Error enum for aarch64 cpu templates
+#[derive(Debug, thiserror::Error)]
+pub enum Error {
+    /// Error getting register
+    #[error("Error getting register {0}: {1}")]
+    GetReg(u64, kvm_ioctls::Error),
```

Comment:
nit: unifying `GetRegister` (like `arch::aarch64::regs::Error`)?
```suggestion
    GetRegister(u64, kvm_ioctls::Error),
```

Reply:
decided to just use arch::aarch64::regs::Error instead of creating a new one